README.md

```diff
@@ -57,103 +57,136 @@ class Apollo apolloBox
 
 ## Getting started
 
-1. Clone this repo
-
 What do you want to do next with this demo app?
 
 ### I want to play around with the public version of the demo
 
-2. Visit the [public Apollo Explorer instance](https://studio.apollographql.com/public/spotify-ev3of9/variant/prod/home) to interact with the graph (No GraphOS account required)
+1. Visit the [public Apollo Explorer instance](https://studio.apollographql.com/public/spotify-ev3of9/variant/prod/home) to interact with the graph (No GraphOS account required)
 
 - [Query the graph](https://studio.apollographql.com/public/spotify-ev3of9/variant/prod/explorer) _(**Spotify account required**) - OAuth workflow with be initiated from Apollo Explorer to login to our Spotify account to run any operation_
   - Try having your Spotify app playing on your phone or desktop and then run [this mutation](https://studio.apollographql.com/public/spotify-ev3of9/variant/prod/explorer?explorerURLState=N4IgJg9gxgrgtgUwHYBcQC4RxighigSwiQAIAFXGAZwTIBtcBPAI1ygGsBZHfI04ADqkSIgA6Ua9Jqw4lBwkWIYs27AMp4UCOUMV6SBKlMYEkAc136AvpZE3hVkFaA)
 - [View the graph's schema](https://studio.apollographql.com/public/spotify-ev3of9/variant/prod/schema/reference)
 
 ### I want to re-create this demo in my GraphOS account
 
-3. Create a [personal API key](https://www.apollographql.com/docs/graphos/org/account/#personal-api-keys)
-4. Run the clone script with the API key
+1. Clone this repo.
+2. Create a [personal API key](https://www.apollographql.com/docs/graphos/org/account/#personal-api-keys).
+3. Run the script below using your personal API key.
 
-```sh
-APOLLO_KEY={YOUR_API_KEY} npm run graphos-demo
-```
+   ```sh
+   AUTH={YOUR_API_KEY} npm run graphos-demo
```

Comment:
The `scipts/demo.ts` was looking for `process.env.AUTH` in line 16. So I changed this accordingly. Should we change the script instead?